### DISCUSSION TITLE
How many time before export data?

### DISCUSSION THREAD

**Discussion by AlbertoTorrisi** [opengento/magento2-gdpr#130] [Q&A] [answered]
When I try to export data (after login) I click to Export data and this is the message: your data export is in pending.
When my data are ready to be downloaded?
Thank you :)

**Comment by thomas-kl1** [comment, marked as answer]
Hi @AlbertoTorrisi,

After you asked to export your data, you must wait for the cron job to run and generate the files.
Once it's done the export will be marked as ready.
If you don't have your cron jobs running, you can manually run them with the bin/magento cron:run command.

Hope I answered your question :)

**Reply by AlbertoTorrisi** [reply]
Thank you for the answer!
Yes, thank you works perfectly!

**Reply by thomas-kl1** [reply]
@AlbertoTorrisi please mark my response as answer so it could help other users :)
